.github/stale.yml

```diff
@@ -6,6 +6,9 @@ daysUntilStale: 14
 # Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
 daysUntilClose: 7
 
+onlyLabels:
+  - "pending author feedback"
+
 # Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
 exemptLabels:
   - pinned
```

Comment:
This configuration is confusing. We have an explicit exemption and an explicit inclusion. Shouldn't there be only a single inclusion? Are `exemptLabels` ignored in this case?